Source/Core/VideoBackends/D3D/D3DBoundingBox.cpp

```diff
@@ -2,18 +2,29 @@
 // Licensed under GPLv2+
 // Refer to the license.txt file included.
 
-#include "VideoBackends/D3D/D3DBoundingBox.h"
+#include <algorithm>
+#include <array>
+
 #include "Common/CommonTypes.h"
 #include "Common/MsgHandler.h"
+#include "VideoBackends/D3D/D3DBoundingBox.h"
 #include "VideoBackends/D3D/D3DState.h"
 #include "VideoBackends/D3DCommon/D3DCommon.h"
 #include "VideoCommon/VideoConfig.h"
 
 namespace DX11
 {
+enum : u32
```

Comment:
`static constexpr u32 NUM_BBOX_VALUES = 4;` (thanks to constexpr, there's no need for the enum trick to get true compile-time constants :D)